packages/components/src/mobile/clipboard/index.native.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext, useCallback, useState } from '@wordpress/element';
+
+export const ClipboardContext = createContext( {} );
+const { Provider, Consumer } = ClipboardContext;
+export { Consumer as ClipboardConsumer };
+
+export default function ClipboardProvider( { children } ) {
+	const [ clipboard, setClipboard ] = useState();
+	const updateClipboard = useCallback( ( clipboardUpdate ) => {
```

Comment:
Isn't `setClipboard` going to be the same reference on each call? In that case we don't need that `updateClipboard` method I think

Reply:
I updated this code, removing the provider/context approach